plugins/blocks/developedBy/templates/block.tpl

```diff
@@ -8,6 +8,10 @@
  * Common site sidebar menu -- "Developed By" block.
  *}
 <div class="pkp_block block_developed_by">
+	<h2 class="title">
```

Comment:
Same here: add the `.pkp_screen_reader` class.

Reply:
Done!